Vagrantfile

```diff
@@ -160,6 +173,20 @@ Vagrant.configure(2) do |config|
     mount_folders(override, [])
   end
 
+  config.vm.provider "libvirt" do |libvirt, override|
+      # Default box with libvirt support
+      unless ENV.has_key?('VAGRANT_BOX')
+        override.vm.box = "generic/ubuntu2004"
```

Comment:
Should we move the regular base image over to `generic/ubuntu2004` instead of using `bento/ubuntu-20.04`? The generic project doesn't look like it'll support arm64 anytime soon so parallels with arm64 will always need an alternative, but that doesn't stop non-arm64 people.

If we want to keep using bento as our base image, we should also look to create an issue on their tracker seeing if they'd be open to adding `libvrt` boxes as that would then allow us to drop the usage of `generic/ubuntu2004` here.